aten/src/ATen/native/Convolution.cpp

```diff
@@ -182,9 +182,9 @@ static void check_input_shape_forward(const at::Tensor& input,
              "Given transposed=", transposed, ", weight of size ", weight.sizes(),
              ", expected input", input.sizes(), " to have ", weight.size(0),
              " channels, but got ", input.size(1), " channels instead");
-    AT_CHECK(!bias.defined() || (bias.ndimension() == 1 && bias.size(0) == weight.size(1) * groups),
```

Comment:
Ah I'm sorry. I was wrong. The original code is correct because the weight for convT is `[group * in_channel_per_group, out_channel_per_group, k0, k1]`. Sorry about it... I don't think this patch is needed.